src/app/redux/UserSaga.js

```diff
@@ -212,186 +228,235 @@ function* usernamePasswordLogin2({
     }
 
     let private_keys;
-    try {
-        const private_key = PrivateKey.fromWif(password);
-        login_wif_owner_pubkey = private_key.toPublicKey().toString();
-        private_keys = fromJS({
-            posting_private: isRole('posting', () => private_key),
-            active_private: isRole('active', () => private_key),
-            memo_private: private_key,
-        });
-    } catch (e) {
-        // Password (non wif)
-        login_owner_pubkey = PrivateKey.fromSeed(username + 'owner' + password)
-            .toPublicKey()
-            .toString();
-        private_keys = fromJS({
-            posting_private: isRole('posting', () =>
-                PrivateKey.fromSeed(username + 'posting' + password)
-            ),
-            active_private: isRole('active', () =>
-                PrivateKey.fromSeed(username + 'active' + password)
-            ),
-            memo_private: PrivateKey.fromSeed(username + 'memo' + password),
+    if (!useKeychain) {
```

Comment:
A better visual diff here:

[usersagadiff.pdf](https://github.com/steemit/condenser/files/2728887/usersagadiff.pdf)

Reply:
Better diff using ?w=1 in query param.

https://github.com/steemit/condenser/pull/3095/files?w=1#diff-9aad451f6da41ec1ad5dc40338a06da2